packages/trackerless-network/generated/packages/dht/protos/DhtRpc.client.ts

```diff
@@ -336,6 +337,10 @@ export interface IConnectionLockRpcClient {
      * @generated from protobuf rpc: gracefulDisconnect(dht.DisconnectNotice) returns (google.protobuf.Empty);
      */
     gracefulDisconnect(input: DisconnectNotice, options?: RpcOptions): UnaryCall<DisconnectNotice, Empty>;
+    /**
+     * @generated from protobuf rpc: setPrivate(dht.SetPrivateRequest) returns (google.protobuf.Empty);
+     */
+    setPrivate(input: SetPrivateRequest, options?: RpcOptions): UnaryCall<SetPrivateRequest, Empty>;
```

Comment:
These changes are not part of this PR?

Reply:
These are unused parts of the generated DHT proto files in the trackerless network

Reply:
Added as other changes in the description

Reply:
Made a ticket about removing unused generated code from the higher layers